src/ContextualPopup/ContextualPopup.js

```diff
@@ -222,6 +229,17 @@ module.exports = kind(
 	initComponents: function () {
 		this.createChrome(this.tools);
 		Popup.prototype.initComponents.apply(this, arguments);
+		this.syncProperties();
+	},
+
+	/**
+	* Sync spotlightModal and modal properties on creation time.
+	*
+	* @private
+	*/
+	syncProperties: function () {
```

Comment:
Also, let's make sure we document this behavior in the public-facing documentation, either in the description of the kind, or in the description of the published `spotlightModal` property. It might not hurt to explicitly document the `modal` property as well so that it is clear that the default value of both properties is `false`, with some details on how they are synced and in what direction.